src/domain/discounts/details/conditions/edit-condition/condition-table-actions.tsx

```diff
@@ -0,0 +1,69 @@
+import clsx from "clsx"
+import React from "react"
+import { useContext } from "react"
+import Button from "../../../../../components/fundamentals/button"
+import PlusIcon from "../../../../../components/fundamentals/icons/plus-icon"
+import { LayeredModalContext } from "../../../../../components/molecules/modal/layered-modal"
+import { useAddConditionsModalScreen } from "./add-conditions-screens"
+import { useEditConditionContext } from "./edit-condition-provider"
+
+const ExistingConditionTableActions = ({
```

Comment:
Sure, I was just trying to avoid long component name, but I think you make a reasonable point 👍

Reply:
I was maybe a bit hasty here. We've used Conditions earlier, so to avoid having to refactor all existing use in this PR, I will keep them as is, if that's cool with you.

Reply:
alright 👍 to keep in mind for later